build/images/ovs/Dockerfile.windows

```diff
@@ -12,42 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG NANOSERVER_VERSION
-
-FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as ovs-base
-
-ARG WIN_OVS_VERSION
-
-# Sets the shell to be PowerShell 
-SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
-
-ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
-
-# Install VC for ovs
-RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
-    Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait; \
-    rm C:\vc_redist.x64.exe;
-
-# Download and extract OVS
-RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip; \
-    Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\; \
-    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;
-
-# Download, extract OpenSSL and copy libraries
-RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
-    Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\openssl; \
-    Remove-Item -Path 'C:\openssl-1.0.2u-x64_86-win64.zip';
-
-FROM mcr.microsoft.com/powershell:lts-nanoserver-${NANOSERVER_VERSION} as windows-ovs
-SHELL ["cmd", "/S", "/C"]
-
-# Switch to a privileged user
-USER ContainerAdministrator
-
-COPY --from=ovs-base /Windows/System32/vcruntime140.dll /Windows/System32/
-COPY --from=ovs-base /openssl/libeay32.dll /Windows/System32/
-COPY --from=ovs-base /openssl/ssleay32.dll /Windows/System32/
-
-COPY --from=ovs-base /openvswitch /openvswitch/
-
-RUN setx /m PATH "C:/openvswitch/usr/bin;C:/openvswitch/usr/sbin;%PATH%"
+FROM --platform=linux/amd64 ubuntu:22.04 as antrea-windows-builder
```

Comment:
Do we still want to keep a dedicated base image for this? It is just copying files.
I feel like the same rationale as for the base image could be applied here.

Reply:
Do you mean antrea-ovs is also removed after switching to hpc?

Reply:
With some tests, I feel it is better to maintain a base image for Windows dependencies, like CNI and OVS, which are independent from Antrea changes. Otherwise, we will always download the related files when building antrea images, and it may introduce the possibility of errors because of network download issues. e.g., Windows build failed because this error, http://10.176.27.169:8080/job/antrea-windows-containerd-networkpolicy-for-pull-request/487/consoleFull

```
#17 ERROR: process "/bin/sh -c wget -q -O - [https://github.com/containernetworking/plugins/releases/download/${CNI_BINARIES_VERSION}/cni-plugins-windows-amd64-${CNI_BINARIES_VERSION}.tgz](https://github.com/containernetworking/plugins/releases/download/$%7BCNI_BINARIES_VERSION%7D/cni-plugins-windows-amd64-$%7BCNI_BINARIES_VERSION%7D.tgz) | tar xz -C /go/k/antrea/cni $CNI_PLUGINS" did not complete successfully: exit code: 2
------
 > [antrea-build-windows 7/7] RUN wget -q -O - https://github.com/containernetworking/plugins/releases/download/v1.4.0/cni-plugins-windows-amd64-v1.4.0.tgz | tar xz -C /go/k/antrea/cni ./host-local.exe:
#17 36.36 
#17 36.36 gzip: stdin: unexpected end of file
#17 36.36 tar: Child returned status 1
#17 36.36 tar: Error is not recoverable: exiting now
------
```

Reply:
> Do you mean antrea-ovs is also removed after switching to hpc? It may introduce build problems when we need to change OVS, e.g., bump up OVS version for bugfix. @antoninbas @XinShuYang

IMO I prefer to keep the antrea-ovs image, as this avoids the need to upload multiple Antrea images when updating the same version of OVS across multiple Antrea releases. Currently, we only need to update the tag of the antrea-ovs image in the manifest, which makes it more convenient for us to perform OVS upgrade tests. Regarding the reason for continuing to use the OVS version as the tag for the antrea-ovs image, I also stated my reasons in https://github.com/antrea-io/antrea/pull/6325#discussion_r1601296629.

Reply:
Ok. Let me resolve this and follow-up on the other thread.